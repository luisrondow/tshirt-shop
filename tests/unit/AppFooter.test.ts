import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'

import { AppFooter } from '#components'

describe('AppFooter component', () => {
  it('should render correctly', () => {
    const wrapper = mount(AppFooter)

    expect(wrapper.vm).toBeTruthy()
    expect(wrapper.element).toMatchSnapshot()
  })

  it('should render the logo', () => {
    const wrapper = mount(AppFooter)

    expect(wrapper.find('h1').exists()).toBe(true)
    expect(wrapper.find('h1').text()).toEqual('Â© 2024 TSHIRT SHOP')
  })
})
