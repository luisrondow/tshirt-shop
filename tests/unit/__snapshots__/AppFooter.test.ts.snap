// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AppFooter component > should render correctly 1`] = `
<footer
  class="flex w-full items-center justify-center bg-black p-2"
>
  <h1
    class="font-serif text-2xl font-semibold text-white"
  >
    Â© 2024 TSHIRT SHOP
  </h1>
</footer>
`;
